### PR TITLE
Theme slaughter update

### PR DESCRIPTION
**Pull request by local-minimum**
* Unified style with rest of site.
* Hotkeys for yes and no (not flag because people shouldn't abuse it) #1381, #343 ? 
* Colors on buttons, recent vote history and chart match.
* You can change your mind for a vote by clicking on it. #1059, #325 
* CSS animations to show what is updating and happening.
* Code cleaning
* Pie-chart supporting custom color definitions.

I haven't seen #332 so maybe close that one too.

ld-theme-slaughter2

### CODE REVIEW

File: src/com/content-event/event-slaughter.less [hunk 1]
```diff
@@ -41,22 +45,42 @@
...
-			border-radius: 2.0em;
+			// border-radius: 2.0em;
 			font-weight: 700;
 			outline: none;
+			color: @COL_L;
 
-			&.-green {
-				background:#4C4;
-				color:#8F8;
+			&.-yes{
```
Comment (mikekasprzak) [inline src/com/content-event/event-slaughter.less]: space ` {`

**Comment by mikekasprzak** [comment]
I'm not seeing where the zoom effect is being triggered. Let me know what's causing it. We're going to have to tweak it to avoid excessive repaints.

**Comment by local-minimum** [comment]
That's strange. Just tested on FF and Chrome and they work fine. But lets take that in a chat. The load on the client should be fine. I'm running it on a not too cool notebook while also having the server running on it and I don't notice any issues.

**Comment by mikekasprzak** [comment]
This is good, but for reference #332 is still there. To reproduce you need to vote on some themes and pay close attention to the order, then refresh. The order breaks.

**Pull request merged**